tests/components/freebox/test_sensors.py

```diff
@@ -0,0 +1,32 @@
+"""Tests for the Freebox sensors."""
+from unittest.mock import Mock
+
+from homeassistant.components.freebox.const import DOMAIN
+from homeassistant.const import CONF_HOST, CONF_PORT
+from homeassistant.core import HomeAssistant
+from homeassistant.setup import async_setup_component
+
+from .const import DATA_STORAGE_GET_RAIDS, MOCK_HOST, MOCK_PORT
+
+from tests.common import MockConfigEntry
+
+
+def bool_to_binary_sensor(value):
+    """Convert the mock data (bool) to the expected BinarySensor state string."""
+    return "on" if value else "off"
+
+
+async def test_sensors(hass: HomeAssistant, router: Mock) -> None:
+    """Test sensors button."""
```

Comment:
Is the docstr correct?

Reply:
No, and the module name neither, i think i messed up my commits at some point, i'll fix this

Reply:
The module name should be `test_binary_sensor.py`, because you only added tests for this part.

Reply:
We can always add tests for other modules in a follow up PR.